python/cudaq/kernel/ast_bridge.py

```diff
@@ -104,6 +104,90 @@ def __init__(self, *args, **kwargs):
         RuntimeError.__init__(self, *args, **kwargs)
 
 
+class PyStack(object):
```

Comment:
I expect given the amount of churn in this file that this will all need to be redone for the python rewrite.